{"uri":"file:///home/tarcisio/dev/phpfin/vendor/symfony/console/Exception/CommandNotFoundException.php","root":{"kind":0,"name":"","children":[{"kind":512,"name":"Symfony\\Component\\Console\\Exception","location":[11,0,11,46]},{"kind":1,"name":"Symfony\\Component\\Console\\Exception\\CommandNotFoundException","location":[18,0,42,1],"associated":[{"kind":1,"name":"InvalidArgumentException"},{"kind":2,"name":"Symfony\\Component\\Console\\Exception\\ExceptionInterface"}],"description":"Represents an incorrect command name typed in the console.","children":[{"kind":16,"name":"$alternatives","modifiers":4,"location":[20,12,20,25],"scope":"Symfony\\Component\\Console\\Exception\\CommandNotFoundException"},{"kind":32,"name":"__construct","modifiers":1,"location":[28,4,33,5],"scope":"Symfony\\Component\\Console\\Exception\\CommandNotFoundException","children":[{"kind":128,"name":"$message","location":[28,32,28,40],"description":"Exception message to throw","scope":"__construct","type":"string"},{"kind":128,"name":"$alternatives","location":[28,42,28,71],"description":"List of similar defined names","scope":"__construct","value":"array()","type":"array"},{"kind":128,"name":"$code","location":[28,73,28,82],"description":"Exception code","scope":"__construct","value":"0","type":"int"},{"kind":128,"name":"$previous","location":[28,84,28,111],"description":"previous exception used for the exception chaining","scope":"__construct","value":"null","typeSource":1,"type":"Exception"}]},{"kind":32,"name":"getAlternatives","modifiers":1,"location":[38,4,41,5],"scope":"Symfony\\Component\\Console\\Exception\\CommandNotFoundException","type":"array"}]}]}}